software/mk/Makefile.builddefs

```diff
@@ -295,8 +296,24 @@ $(BSG_MANYCORE_LIB): $(BSG_MANYCORE_LIB_OBJS)
 
 
 SKIP_DRAM_INSTRUCTION_LOAD ?= 0
+HOST ?= x86
+
+# BlackParrot-specific arguments
+ifeq ($(HOST),bp)
+NBF_SCRIPT   ?= $(NBF_BP_PY)
+BP_MEM_IMAGE ?= prog.mem
+BP_DRAM_BASE ?= 0x82000000
+BP_DRAM_POD  ?= 0x09
+else
+NBF_SCRIPT   ?= $(NBF_PY)
+BP_MEM_IMAGE ?=
+BP_DRAM_BASE ?=
+BP_DRAM_POD  ?=
+endif
+
 %.nbf: %.riscv
-	python $(NBF_PY) $*.riscv \
+	python $(NBF_SCRIPT) \
+	  $*.riscv \
```

Comment:
Can you make these parameters a separate makefile variable (e.g. $NBF_PARAM), and add the bp specific params if $HOST equals bp? so it doesn't break the existing nbf.py

Reply:
I can do that for the BP-specific parameters. But which script to run is going to remain a separate variable as it is now.